content/zh/faq/distributed-tracing/how-to-support-tracing.md

```diff
@@ -9,7 +9,7 @@ Istio 允许报告服务网格中工作负载到工作负载间通信的追踪 s
 
 - `x-request-id`
 - `x-b3-traceid`
-- `x-b3-spanId`
+- `x-b3-spanid`
```

Comment:
It is recommended not to make changes to zh files in the same PR as en ones. The translation team will sync the zh file with the en change in a separate PR, after the en change has merged.